statistics/statistics.go

```diff
@@ -41,6 +42,7 @@ type Plugin struct {
 	bufferMaxSize int
 	bufferCurSize int
 	bufferIndex   int
+	logger        *log.Logger
```

Comment:
This was recommended from a review Balaji forwarded me from Izabella Raulin, Katarzyna Zabrocka, and Kasia.